envoy/server/configuration.h

```diff
@@ -84,6 +84,11 @@ class StatsConfig {
    * @return bool indicator to flush stats on-demand via the admin interface instead of on a timer.
    */
   virtual bool flushOnAdmin() const PURE;
+
+  /**
+   * When true, enable lazy-init of stats.
```

Comment:
I slightly prefer LazyInit in the code even if we use "Deferred" in the doc, but  either way, within the code we should be consistent.

Reply:
thanks for the catch, use deferred creation here to keep consistency.